config/webpack.test.js

```diff
@@ -87,84 +98,140 @@ module.exports = function (options) {
          *
          * See: https://github.com/webpack/source-map-loader
          */
-        // {
-        //   test: /\.js$/,
-        //   use: ['source-map-loader'],
-        //   exclude: [
-        //     // these packages have problems with their sourcemaps
-        //     helpers.nodeModulePath("mydatepicker"),
-        //     helpers.nodeModulePath("ng2-completer"),
-        //     helpers.nodeModulePath("angular2-flash-messages"),
-        //     helpers.nodeModulePath("ngx-dropdown"),
-        //     helpers.nodeModulePath("ngx-modal"),
-        //     helpers.nodeModulePath("ngx-modal"),
-        //     helpers.nodeModulePath("ng2-dnd")
-        //   ]
-        // },
+        {
+          test: /\.js$/,
+          use: ['source-map-loader'],
+          exclude: [
+            // these packages have problems with their sourcemaps
+            helpers.root('node_modules/rxjs'),
+            helpers.root('node_modules/@angular')
+            //     helpers.nodeModulePath("mydatepicker"),
+            //     helpers.nodeModulePath("ng2-completer"),
+            //     helpers.nodeModulePath("angular2-flash-messages"),
+            //     helpers.nodeModulePath("ngx-dropdown"),
+            //     helpers.nodeModulePath("ngx-modal"),
+            //     helpers.nodeModulePath("ngx-modal"),
+            //     helpers.nodeModulePath("ng2-dnd")
+          ]
+        },
 
-        /**
-         * Typescript loader support for .ts and Angular 2 async routes via .async.ts
-         *
-         * See: https://github.com/s-panferov/awesome-typescript-loader
-         */
         {
           test: /\.ts$/,
           use: [
             {
-              loader: "awesome-typescript-loader",
+              loader: 'ts-loader',
               options: {
-                configFileName: 'tsconfig-test.json'
+                configFile: 'tsconfig-test.json'
               }
             },
-            {
-              loader: "angular2-template-loader"
-            }
-
+            'angular2-template-loader'
           ],
           exclude: [/\.e2e\.ts$/]
         },
 
+        /**
+         * Static analysis linter for TypeScript advanced options configuration
+         * Description: An extensible linter for the TypeScript language.
+         *
+         * See: https://github.com/wbuchwalter/tslint-loader
+         */
+        // {
+        //   test: /\.ts$/,
+        //   enforce: 'pre',
+        //   loader: 'tslint-loader',
```

Comment:
Are these commented lines intentional?

Reply:
I want this to work and don't want to loose it yet. So it is commented till I can get it to function.